tulflow/validate.py

```diff
@@ -49,9 +49,9 @@ def filter_s3_schematron(**kwargs):
                 total_filter_count += 1
                 invalid_csv.writerow({
                     "id": record_id,
-                    "report": etree.tostring(schematron.validation_report),
-                    "record": etree.tostring(record),
-                    "source_file": s3_key
+                    "report": schematron_failed_validation_text(schematron.validation_report),
+                    "record": identifier_or_full_record(record),
+                    "source_file": f"https://s3.console.aws.amazon.com/s3/object/bucket/{s3_key}"
```

Comment:
@bibliotechy, just verifying for my own edification mostly: shouldn't this source_file URL be in the format `S3://bucket-name/key-name`?

Also if not, should we be parsing the bucket as well .. i.e. `f"https://s3.console.aws.amazon.com/s3/object/{bucket}/{s3_key}"`

Reply:
@dkinzer So, if we wanted to link directly to the s3 bucket via `S3://bucket-name/key-name`, we'd also have to either make that file publicly available or presign the URL, which would probably expire before it would be useful in the report. 
So I opted to add the link to the console, which would at least prompt for a login. BUt you are right, bucket should be parsed as well. 
Thanks for catching that

Reply:
@dkinzer Updated